core/src/trezor/lvglui/i18n/keys.py

```
@@ -1829,85 +1837,252 @@
+PASSPHRASE__PIN_ATTACHED = 900
+# You can attach a passphrase to this PIN.
+PASSPHRASE__PIN_NOT_ATTACHED_DESC = 901
+# You can remove this PIN or attach a new Passphrase.
+PASSPHRASE__PIN_ATTACHED_DESC = 902
+# PIN Already Used
+PASSPHRASE__PIN_USED = 903
+# This PIN has already been used. Please try a different one.
+PASSPHRASE__PIN_USED_DESC = 904
+# Hit the Limit
+PASSPHRASE__PIN_HIT_LIMIT = 905
+# You’ve reached the maximum of 30 PINs. Please remove some before adding new
```

Ruff (0.11.9)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)